src/BenchmarkDotNet/ConsoleArguments/CommandLineOptions.cs

```diff
@@ -97,8 +97,8 @@ public class CommandLineOptions
         [Option("ilCompilerVersion", Required = false, HelpText = "Optional version of Microsoft.DotNet.ILCompiler which should be used to run with NativeAOT. Example: \"7.0.0-preview.3.22123.2\"")]
         public string ILCompilerVersion { get; set; }
 
-        [Option("ilcPath", Required = false, HelpText = "Optional IlcPath which should be used to run with private NativeAOT build.")]
-        public DirectoryInfo IlcPath { get; set; }
+        [Option("ilcPackages", Required = false, HelpText = @"Optional path to shipping packages produced by local dotnet/runtime build. Example: 'D:\projects\forks\jkotas\artifacts\packages\Release\Shipping\'")]
```

Comment:
```suggestion
        [Option("ilcPackages", Required = false, HelpText = @"Optional path to shipping packages produced by local dotnet/runtime build. Example: 'D:\projects\forks\runtime\artifacts\packages\Release\Shipping\'")]
```
`artifacts` is subdirectory of runtime in the default setup

Reply:
@jkotas sorry, I was using a copy of your fork to test one of your PRs ;p